lib/Sema/TypeCheckNameLookup.cpp

```diff
@@ -566,7 +572,7 @@ void TypeChecker::performTypoCorrection(DeclContext *DC, DeclRefKind refKind,
   entries.filterMaxScoreRange(MaxCallEditDistanceFromBestCandidate);
 
   for (auto &entry : entries)
-    result.add({ entry.Value, nullptr });
+    result.add({ entry.Value, nullptr, false });
```

Comment:
/*promotedInstanceRef=*/false would be clearer.

Also, a similar "label" for the nullptr parameter would be nice too.